test/integration/framework.go

```diff
@@ -225,6 +228,19 @@ func isFastpathBESS() bool {
 	return os.Getenv(EnvFastpath) == FastpathBESS
 }
 
+func initForwardingPipelineConfig() {
+	p4rtClient, err := providers.ConnectP4rt("127.0.0.1:50001", ReaderElectionID)
+	if err != nil {
+		panic("Cannot init forwarding pipeline config: " + err.Error())
+	}
+	defer providers.DisconnectP4rt()
+
+	_, err = p4rtClient.SetFwdPipe(deviceConfigPath, p4InfoPath, 0)
+	if err != nil {
+		panic("Cannot init forwarding pipeline config: " + err.Error())
+	}
+}
```

Comment:
There should be a way to start `simple_switch_grpc` or `stratum_bmv2` with a pre-loaded pipeline config. However, this is perfectly fine.

Reply:
@daniele-moro yep. from stratum_bmv2 (present in mock-up4 container, used while performing the integration tests):

Flags from stratum/hal/bin/bmv2/main.cc:
    -bmv2_log_level (Log level of Bmv2(trace, debug, info, warn, error, off))
      type: string default: "info"
    -console_logging (Log BMv2 message to console.) type: bool default: true
    -cpu_port (BMv2 port number for CPU port (used for packet I/O))
      type: uint32 default: 64
    -device_id (BMv2 device/node id) type: uint32 default: 1
    -**initial_pipeline (Path to initial pipeline for BMv2 (required for starting
      BMv2)) type: string default: "/etc/stratum/dummy.json"**

we should be able to do that using the `-initial_pipeline` flag